dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -95,12 +93,103 @@ def shell(verbose: bool):
+)
+@click.option('--dev-apt-command', help='The basic command executed before dev apt deps are installed.')
+@click.option(
+    '--dev-apt-deps',
+    help='The basic apt dev dependencies to use when building the images.',
+)
+@click.option(
+    '--runtime-apt-command', help='The basic command executed before runtime apt deps are installed.'
+)
+@click.option(
+    '--runtime-apt-deps',
+    help='The basic apt runtime dependencies to use when building the images.',
+)
+@click.option('--github-repository', help='Choose repository to push/pull image.')
+@click.option('--build-cache', help='Cache option')
+@click.option('--upgrade-to-newer-dependencies', is_flag=True)
+def build_ci_image(
+    verbose: bool,
+    additional_extras: Optional[str],
+    python: Optional[float],
+    additional_dev_apt_deps: Optional[str],
+    additional_runtime_apt_deps: Optional[str],
+    additional_python_deps: Optional[str],
+    additional_dev_apt_command: Optional[str],
+    additional_runtime_apt_command: Optional[str],
+    additional_dev_apt_env: Optional[str],
+    additional_runtime_apt_env: Optional[str],
+    dev_apt_command: Optional[str],
+    dev_apt_deps: Optional[str],
+    runtime_apt_command: Optional[str],
+    runtime_apt_deps: Optional[str],
+    github_repository: Optional[str],
+    build_cache: Optional[str],
+    upgrade_to_newer_dependencies: bool,
+):
+    """Builds docker CI image without entering the container."""
+    from airflow_breeze.ci.build_image import build_image
+
     if verbose:
         console.print(f"\n[blue]Building image of airflow from {__AIRFLOW_SOURCES_ROOT}[/]\n")
-    raise ClickException("\nPlease implement building the CI image\n")
+    build_image(
+        verbose,
+        additional_extras=additional_extras,
+        python_version=python,
+        additional_dev_apt_deps=additional_dev_apt_deps,
+        additional_runtime_apt_deps=additional_runtime_apt_deps,
+        additional_python_deps=additional_python_deps,
+        additional_runtime_apt_command=additional_runtime_apt_command,
+        additional_dev_apt_command=additional_dev_apt_command,
+        additional_dev_apt_env=additional_dev_apt_env,
+        additional_runtime_apt_env=additional_runtime_apt_env,
+        dev_apt_command=dev_apt_command,
+        dev_apt_deps=dev_apt_deps,
+        runtime_apt_command=runtime_apt_command,
+        runtime_apt_deps=runtime_apt_deps,
+        github_repository=github_repository,
+        docker_cache=build_cache,
+        upgrade_to_newer_dependencies=upgrade_to_newer_dependencies,
+    )
+
+
+@option_verbose
+@main.command(name='build-prod-image')
+def build_prod_image(verbose: bool):
+    """Builds docker Production image without entering the container."""
+    if verbose:
+        console.print("\n[blue]Building image[/]\n")
+    raise ClickException("\nPlease implement building the Production image\n")
 
 
 if __name__ == '__main__':
```

Comment:
```suggestion
if __name__ == '__main__':
```suggestion
     BUILD_CACHE_DIR.mkdir(parents=True, exist_ok=True)
```

Also BUILD_CACHE_DIR should be impeorted. This will make sure that `.build` directory always exsists even in just checked-out repository.

Reply:
@potiuk I have resolved this issue. But I faced a circular import error when I added the import part at the beginning of the file. So I have added the import just before mkdir command. Is this correct way of handling or can you suggest better way than this?

Reply:
It's correct!